fast-af/fast-checkout-woocommerce
# Add check to shortcodes

# Description

Add checks in shortcodes for whether or not the Fast checkout buttons should be hidden.

# Testing instructions

1. Add a shortcode: `[fast_cart /]` or `[fast_product /]`
2. Verify that the button shows when it should and is hidden when it should be.

# Checks

@ilkerulutas @brikr @ejcho623 this is ready for a review.
